api/v1/ocsinitialization_types.go

```diff
@@ -29,11 +29,13 @@ type OCSInitializationSpec struct {
 	// should be deployed.
 	// Defaults to false
 	// +optional
+	// +kubebuilder:deprecatedversion:warning="This field has been deprecated and will be removed in future. The ceph tool box can be enabled by setting the enableCephTools field in the storage cluster CR Spec"
```

Comment:
I think we should change this message and point to the storagecluster spec which can be used for the same purpose.